src/driver/RealtimeDatabase/InProcessRealtimeDatabase.ts

```diff
@@ -349,7 +349,20 @@ export class InProcessRealtimeDatabaseRef
                 `Cannot update Firebase Realtime Database path to undefined (${this.path})`,
             )
         }
-        this.db._updatePath(this.path, value)
+        const keys = Object.keys(value)
+        const updatedValue = {}
+        keys.forEach((key) => {
+            const splitKeys = key.split("/")
+            if (splitKeys.length > 1) {
+                // @ts-ignore
```

Comment:
I had to add this because it complains about:
`Element implicitly has an 'any' type because expression of type '"key"' can't be used to index type '{}'.`